flask/app/models.py

```diff
@@ -0,0 +1,48 @@
+from app import db
+from datetime import datetime
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(128), unique=True, nullable=False)
+    first_name = db.Column(db.String(128), nullable=False)
+    last_name = db.Column(db.String(128), nullable=False)
+    posts = db.relationship("CodeFile", backref="author", lazy=True)
+
+    def __init__(self, email, first_name, last_name):
+        self.email = email
+        self.first_name = first_name
+        self.last_name = last_name
+
+    def __repr__(self):
+        return f"User id: {id} email:{email}"
+
+    def profileMap(self):
+        return {
+            "id": self.id,
+            "email": self.email,
+            "first_name": self.first_name,
+            "last_name": self.last_name,
+        }
+
+
+class CodeFile(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(200), nullable=False)
+    date_posted = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    content = db.Column(db.Text, nullable=True)
+    language = db.Column(db.String(50), nullable=False)
+    # image_file = db.Column(db.String(20), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey("user.id"), nullable=False)
+
+    def __init__(self, title, content, language, user_id):
+        self.title = title
+        self.content = content
+        self.language = language
+        self.user_id = user_id
+
+
+db.drop_all()
```

Comment:
For these last couple of rows here, can we write something instead so that we aren't always dropping the tables? Idk if this has something to do with hooking up alembic (which I can help with) or if it's to do with data persistance.